napari/utils/theme.py

```diff
@@ -127,52 +127,48 @@ def to_rgb_dict(self) -> Dict[str, Any]:
 opacity_pattern = re.compile(r'{{\s?opacity\((\w+),?\s?([-\d]+)?\)\s?}}')
 
 
-def decrease(font_size: str, pt: int):
+def decrease(font_size: str, pt: int) -> str:
     """Decrease fontsize."""
     return f"{int(font_size[:-2]) - int(pt)}pt"
 
 
-def increase(font_size: str, pt: int):
+def increase(font_size: str, pt: int) -> str:
     """Increase fontsize."""
     return f"{int(font_size[:-2]) + int(pt)}pt"
 
 
-def darken(color: Union[str, Color], percentage=10):
-    if isinstance(color, str) and color.startswith('rgb('):
-        color = literal_eval(color.lstrip('rgb(').rstrip(')'))
-    else:
-        color = color.as_rgb_tuple()
+def _parse_color_as_rgb(color: Union[str, Color]) -> Tuple[int, int, int]:
+    if isinstance(color, str):
+        if color.startswith('rgb('):
+            return literal_eval(color.lstrip('rgb(').rstrip(')'))
```

Comment:
Comment on the original code. I find it strange to use lstrip/rstrip as those are for repeated patterns and mean rstrip any of `r`, `g`, `b` or `(`. I think `color[4:-1]` or [removeprefix/removesuffix](https://docs.python.org/3/library/stdtypes.html#str.removeprefix) once we are 3.9+.